src/main/java/com/client/core/resttrigger/controller/AbstractRestTriggerController.java

```diff
@@ -31,22 +33,24 @@
     private final Class<E> type;
     private final List<TriggerValidator<E, H, T>> triggerValidators;
 
-    protected final BullhornData bullhornData;
+    @Autowired
+    protected BullhornData bullhornData;
```

Comment:
Why not final anymore? Can't you `@Autowire` final dependencies?

Reply:
You cannot `@Autowired` final member variables.
HOWEVER,
You can `@Autowired` on a class constructor and then set a final member variable.